src/Grids/latitude_longitude_grid.jl

```diff
@@ -323,13 +323,13 @@ end
 @inline znodes(grid::LatitudeLongitudeGrid, ZL::Face  ; with_halos=false) = with_halos ? grid.zᵃᵃᶠ : view(grid.zᵃᵃᶠ, interior_indices(typeof(ZL), topology(grid, 3), grid.Nz))
 @inline znodes(grid::LatitudeLongitudeGrid, ZL::Center; with_halos=false) = with_halos ? grid.zᵃᵃᶜ : view(grid.zᵃᵃᶜ, interior_indices(typeof(ZL), topology(grid, 3), grid.Nz))
 
-@inline xnode(i, grid::LatitudeLongitudeGrid, XL::CoF) = xnodes(grid, XL)[i]
-@inline ynode(j, grid::LatitudeLongitudeGrid, YL::CoF) = ynodes(grid, YL)[j]
-@inline znode(k, grid::LatitudeLongitudeGrid, ZL::CoF) = znodes(grid, ZL)[k]
+@inline xnode(i, grid::LatitudeLongitudeGrid, XL::CoF; kwargs...) = xnodes(grid, XL; kwargs...)[i]
+@inline ynode(j, grid::LatitudeLongitudeGrid, YL::CoF; kwargs...) = ynodes(grid, YL; kwargs...)[j]
+@inline znode(k, grid::LatitudeLongitudeGrid, ZL::CoF; kwargs...) = znodes(grid, ZL; kwargs...)[k]
```

Comment:
We don't want to use this pattern for these low-level functions that are supposed to be inlined on the GPU